docstore/docstore.go

```diff
@@ -136,8 +136,9 @@ func (l *ActionList) add(a *Action) *ActionList {
 // AlreadyExists is returned if it does. (See gocloud.dev/gcerrors for more on error
 // codes.)
 //
-// If the document doesn't have key fields, key fields with unique values will be
-// created and doc will be populated with them.
+// If the document doesn't have key fields, or the key fields is empty, meaning
```

Comment:
is => are
meaning as => meaning
and any => or any

Reply:
Done.